rubyqzhao/jaqb-attendance-app
# Feature/admin app

Admin app view courses, view users, and add/delete courses functionality.

To test:
1. Start up the app.
2. On the page, you should be able to see a list of the courses that are currently in the database, a button for User Privileges, a section for "Add Courses", and another for "Delete Courses".
3. Go to the User Privileges page to check that it displays a list of all the users registered in the db.
3. To add a course, enter the required information into the form and press Add. If the add was successful, the page should display "Submitted". Then you can check the db to see if it was successfully added. If you go back to the index page and refresh, the list of courses should also update.
4. To delete a course, enter the course code (make sure the string matches exactly) and press Delete. The page should display "deleted". Then you can check the db to see if it was successfully deleted. If you go back to the index page and refresh, the list of courses should also update.
